tests/unit/backup/backends/test_v2.py

```diff
@@ -29,7 +30,7 @@ def test_schema_backup_v2_roundtrip(tmp_path: Path) -> None:
     topic_name = "a-topic"
     partition_index = 123
     records = (
-        ConsumerRecord(
+        StubMessage(
```

Comment:
See `test.utils.StubMessage`, as `confluent_kafka.Message` cannot be user-instantiated